content/22.vaccines.md

```diff
@@ -470,6 +469,76 @@ Similar to BNT162b2, the vaccine was associated with mild-to-moderate adverse ef
+Investigation of viral dynamics of asymptomatic infection in early 2020 indicated that asymptomatic patients continued to shed the virus for a duration similar to that of symptomatic patients [@doi:10.1016/j.ijid.2020.05.030] (although viral shedding should not be conflated with contagiousness without further investigation [@individual-pathogenesis]).
+Another study found viral load to be higher in the nasopharyngeal/oropharyngeal samples of asymptomatic patients compared to symptomatic patients hospitalized due to symptoms and/or known exposure [@doi:10.1007/s15010-020-01548-8].
+However, the sample size in both of these studies was small, and a larger study found higher viral load in symptomatic than asymptomatic cases [@doi:10.3390/v13020304] along with a systematic review finding a reduced probability of asymptomatic transmission [@doi:10.3138/jammi-2020-0030].
+While far from conclusive, these studies therefore suggest that asymptomatic cases still cary a risk of transmitting SARS-CoV-2.
+
+One important consideration is therefore how likely vaccinated individuals are to develop asymptomatic SARS-CoV-2.
+Considering asymptomatic cases is necessary to establish a more complete picture of efficacy with respect to spread.<!-- To Do: Possible figure indicating pyramid of risk with and without vaccines?-->
+Routine testing of healthcare workers in California who had received an mRNA vaccine revealed slightly higher rates of absolute risk for testing positive than those identified in the phase 2/3 trials, although the extent to which asymptomatic infection influenced these numbers was not investigated [@doi:10.1056/NEJMc2101927].
+Another study analyzed the results of COVID-19 screening tests administered to asymptomatic individuals prior to receiving certain medical services at the Mayo Clinic in several locations across the United States.
+This study found patients who had received two doses of an mRNA vaccine to be 73% less likely to have asymptomatic COVID-19 than patients who had received zero doses [@doi:10.1093/cid/ciab229/6167855].
+Because this study began on December 17, 2020, a date selected to coincide with the first day vaccines were available at the Mayo Clinic, this number may underestimate the efficacy of the vaccines given that many people eligible for early vaccination were at increased risk for exposure (e.g., healthcare workers and residents of long-term care facilities) [@doi:10.1093/cid/ciab229/6167855].
+In Israel, a longitudinal study of nearly 12,000 healthcare workers found that of 5,372 fully vaccinated with Pfizer/BioNTech BNT162b2, eight developed symptomatic COVID-19 (BNT162b2 (.15%) and 19 developed asymptomatic COVID-19 (.35%) [@doi:10.1001/jama.2021.7152].
+While the study itself analyzed the efficacy of the vaccine based on person-days, these findings also suggest that many or even the majority of SARS-CoV-2 infections in vaccinated individuals are likely to be asymptomatic.
```

Comment:
This is interesting. I hadn't thought about the effect of vaccinated individuals contributing to the spread of the virus, given the % of positives being much lower in that population. Do you think this will have a big impact in the spread?

Reply:
Just read ahead and it looks like you answered this. I'm not sure if there is a way to quantify impact, but the bottom line being that there is _some_ contribution from vaccinated individuals, and while small, still we should be cautious because vaccinated individuals can still transmit the virus but how much more compared to unvaccinated individuals is still unknown...is that the takeaway?